### ISSUE TITLE
False positive with `require-input-label` rule and hidden inputs

### ISSUE THREAD

**Issue by bmish** [ember-template-lint/ember-template-lint#1538]
Hidden inputs should be ignored in the [require-input-label](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/require-input-label.md) rule right?

```hbs
<input type="hidden" name="data" value={{this.someData}}>
```
